core/tests/protractor.conf.js

```diff
@@ -312,7 +312,7 @@ exports.config = {
     // a failed run.
     // TODO(sll): Switch this option on by default, once the above issues are
```

Comment:
Will it be fine to remove the TODO comment here?

Reply:
done